article.html

```diff
@@ -1,95 +0,0 @@
-<!DOCTYPE html>

```

Comment:
I removed this plain article file as it was just a default template for all future articles. I can use current article files and copy and paste their template instead.